website/guides/sgb_border.md

```diff
@@ -0,0 +1,178 @@
+- read `P1.0`-`P1.3`, and check if it either
+	* has changed
+	* isn't `%1111`
+- repeat a few times and branch somewhere if the test keeps failing
+
+Let's go over an example snippet ([source](https://github.com/zlago/snek-gbc/blob/baef0369f57d2b0d58316cb1c28c6cc22475a6c9/code/init.sm83#L167-L196)):
+
+```sm83asm
+; test for SGB
+xor a /* first it enables STAT interrupt, since snek-gbc happens to just have a `reti` as the handler */
+ldh [rLYC], a
+ld a, STATF_LYC
+ldh [rSTAT], a
+ld a, IEF_STAT
+ldh [rIE], a
+ei
+	; wait for SGB
+	ld b, 12 /* You must wait 12 or more frames before trying to send a packet */
+	:halt
+	dec b
+	jr nz, :-
+	; enable multi
+	ld hl, Packets.mlt /* send a `MLT_REQ` */
+	call Packet
+	rept 4 /* then wait 4 more frames, since the SNES won't "read" the packet instantly */
+		halt
+	endr
+	; check if SGB responds
+		/* and now We actually try to detect the SGB
+		setting `P1.5` to low then high advances the "read player"
+		setting `P1.4` and `P1.5` high will make the SGB return which player is currently selected in `P1.0` and `P1.1` */
+		lb bc, 5, LOW(rP1) /* b is loaded with 5 which is how many times We try to check if the player changes to anything but P1 */
+		:ld a, P1F_4 /* try to advance player */
+		ldh [c], a
+		ld a, P1F_4|P1F_5 /* then set P1 to return the current player */
+		ldh [c], a
+		ldh a, [c]
+		dec b
+		jp z, .init ; give up /* if 5 (4 actually) attempts fail, assume this isn't an SGB */
+		cp $ff /* `P1.6` and `P1.7` always return %1, We set `P1.4` and `P1.5` to %1, and DMG, or SGB player 1, return %1111 in `P1.0`-`P1.3` */
+		jr z, :- ; /* try again if detection fails */
+```
+
+If you adopt this, remember to change `.init` to wherever code should jump to if it's _not_ on SGB, and put code meant to run on SGB after the snippet.
+
+### SGB detection notes
+
+- It would be a good idea to save somewhere in RAM whether the game is running on an SGB capable device or not, such that if you wish to change the border mid-gameplay, you won't have to perform SGB detection again
+
+- **If you wish to only use 1 controller for the game, you will have to send another `MLT_REQ` to disable multiplayer** (`$89, $00`)
+
+- You can change the waitloop to not use interrupts, or to use di+halt instead
+
+## Border limitations
+
+An SGB border has:
+
+- 255 tiles + 1 transparent tile (preferably tile #0)
+- 3 palettes of 15 (+ 1 transparent) colors, (up to 45 solid colors total)
+- a 256x224px tilemap (there's a bit more to this, see [notes](#notes))
+
+## Converting borders
+
+With a recent version of [superfamiconv](https://github.com/Optiroc/SuperFamiconv):
+
+```sh
+superfamiconv -v -i input.png -p output.pal -t output.4bpp -P 4 -m output.pct -M snes --color-zero 0000ff -B 4
+```
+
+- `--color-zero` should be the color _you_ used for transparency, in my case it was blue.
+	* it can also be set to `00000000` to use the actual transparent color, however, this may cause some issues..
+- `-v` is optional
```

Comment:
```suggestion
- `--color-zero` should be the color that your image for transparency, in my case it was blue.
	* If your image has an alpha channel, it can be set can also be set to `00000000` to use the actual transparent color; however, this may cause some issues.
- `-v` is optional, for showing details of the conversion process
```